### PR TITLE
Automate Releases (step 1)

### PR DESCRIPTION
**Pull request by briantist**
Building out GitHub actions for automating releases

### CODE REVIEW

**Comment by briantist** [comment]
It appears there may be no way to test Actions submitted in a PR from a fork, unless they happen to use triggers that fire on a PR (like pull request and push).

I'm going to switch to trying to push a branch up this repo directly, instead of my fork. I think that might work (based on the test actions I can see from Rohit that appear to be on a branch).

**Comment by briantist** [comment]
I'm changing to draft but reviewers who were tagged will probably need to manually unsubscribe if you don't want notifications.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+          fi
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ env.VER_TAG }}
+          release_name: v${{ env.VER_TAG }}
```
Comment (zturechek) [inline .github/workflows/release.yml]: nit request to remove the preceding `v` to keep with the current pattern for releases

**Review by zturechek — APPROVED** [review]
All your automation are belong to us

**Pull request merged**